.travis_scripts/create_feedstocks.py

```diff
@@ -121,8 +121,11 @@ def write_token(name, token):
     print('Calculating the recipes which need to be turned into feedstocks.')
     with tmp_dir('__feedstocks') as feedstocks_dir:
         feedstock_dirs = []
-        for num, (recipe_dir, name) in enumerate(list_recipes()):
+        recipe_list = list(list_recipes())
+        for num, (recipe_dir, name) in enumerate(recipe_list):
             if num >= 7:
+                if len(recipe_list) > num:
```

Comment:
Isn't this always true?

Reply:
Good point. Fixing with PR ( https://github.com/conda-forge/staged-recipes/pull/5418 ).